### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by tchasinga**
Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
Make sure that you used correct [GitHub Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
Make sure that you documented your work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
HTML/CSS requirements
Follow our list of the [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
JavaScript requirements
Follow our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
Project requirements
Set up a new project with webpack that is based on the webpack exercise you have already completed.
Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (<div> or <ul> element). The index.html file must be set as a template using the HTML Webpack Plugin.
Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three keys:
description [string].
completed [bool].
index: [number].
Write a function to iterate over the tasks array and populate an HTML list item element for each task.
On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

### CODE REVIEW

**Review by 1GORDON — COMMENTED** [review]
##
## Hi @tchasinga ✋

Good job so far!

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

### This project will be marked as Invalid and won't affect the number of your weekly reviews.

- [ ] You have submitted a project with failing linter checks. Kindly ensure your linter checks are passing and are in green.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @tchasinga , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not use the correct readme template 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+module.exports = {
+  mode: 'development',
+  entry: {
+    index: path.resolve(__dirname, './src/index.js'),
+  },
+  output: {
+    path: path.resolve(__dirname, './dist'),
+    filename: 'Running.js',
+    // clean: true,
```
Comment (Zuhaib042) [inline webpack.config.js:12]: - [ ] Kindly consider removing this commented line as having commented code in our repo makes it unprofessional.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @tchasinga 
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✅ Descriptive PR
✅ Linters passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tchasinga** [comment]
@Zuhaib042 hi for your second require change , 
those class are not really the same , and if I'm trying to remove or change them it will take more time , so as long as it work please keep it like that ... and as you can this project is really different than the previous one , 

thanks for your understanding qu'ALHA VOUS PROJECT

**Comment by tchasinga** [comment]
@Zuhaib042  I need to sleep also from morning up to now without breaking ⌚

**Pull request merged**